recipes/traittypes/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set version = "0.0.4" %}
+{% set md5 = "6378e67cd60c8f8f13bdae2b2a2ef307" %}
+
+package:
+    name: traittypes
+    version: {{ version }}
+
+source:
+    fn: traittypes-{{ version }}.tar.gz
+    url: https://pypi.io/packages/source/t/traittypes/traittypes-{{ version }}.tar.gz
+    md5: {{ md5 }}
+
+build:
+    number: 0
+    script: python setup.py install
+
+requirements:
+    build:
+        - python
+    run:
```

Comment:
Are you meaning Travis CI? If so, they are having some infrastructure issues. It seems CircleCI built ok. Not sure what is up with AppVeyor yet.